rpc/wrtc_call_queue_mongodb.go

```diff
@@ -1119,6 +1119,7 @@ func (queue *mongoDBWebRTCCallQueue) RecvOffer(ctx context.Context, hosts []stri
 						recvOfferCtx,
 						bson.D{
 							{webrtcCallIDField, callReq.ID},
+							{webrtcCallAnsweredField, false},
```

Comment:
I'd still like to know why we expect this second attempt to "do good". For example, if the first query failed because the "[caller error field](https://github.com/viamrobotics/goutils/pull/283/files#diff-c7273d63cbd81a307511aef73b2baa73baa9de7e566898c4ce7237f3ed192de0R1085)" had a value -- is that a case where this second chance will yield a better outcome?

Reply:
It's less a second chance and more so the case that the code is now on the "train" of collection events. Each one that comes in gives the code a shot to claim the call request. That is, it's a race among the other answerers for that host to claim the document/call.